docs/source/reference/numpysupported.rst

```diff
@@ -182,6 +182,7 @@ floating-point and complex numbers:
 * :func:`numpy.linalg.eigvalsh` (only the first argument).
 * :func:`numpy.linalg.inv`
 * :func:`numpy.linalg.lstsq`
+* :func:`numpy.linalg.matrix_power`
```

Comment:
Fixed in https://github.com/stuartarchibald/numba/commit/f3c147743efac1a86bd389faf8e9f9becbbbd978 I'll PR once this is merged.